From: Andreas Gruenbacher <agruen@suse.de>
Subject: Hard-code the RPM release number; don't get it from .config

The release number really belongs to the source tree, not the
configuration file. We had at least one error report where people
did something like cloneconfig, and ended up with a release number
that didn't match the source tree.

Signed-off-by: Andreas Gruenbacher <agruen@suse.de>

 Makefile |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletion(-)

Index: linux-2.6.13/Makefile
===================================================================
--- linux-2.6.13.orig/Makefile	2005-10-07 21:01:07.000000000 -0700
+++ linux-2.6.13/Makefile	2005-10-07 21:01:12.000000000 -0700
@@ -157,8 +157,11 @@
 LOCALVERSION = $(subst $(space),, \
 	       $(shell cat /dev/null $(localver)) \
 	       $(patsubst "%",%,$(CONFIG_LOCALVERSION)))
+ifneq ($(wildcard $(srctree)/rpm-release),)
+RPM_RELEASE := -$(shell cat $(srctree)/rpm-release)
+endif
 
-KERNELRELEASE=$(VERSION).$(PATCHLEVEL).$(SUBLEVEL)$(EXTRAVERSION)$(LOCALVERSION)
+KERNELRELEASE=$(VERSION).$(PATCHLEVEL).$(SUBLEVEL)$(EXTRAVERSION)$(RPM_RELEASE)$(LOCALVERSION)
 
 # SUBARCH tells the usermode build what the underlying arch is.  That is set
 # first, and if a usermode build is happening, the "ARCH=um" on the command
